src/UserSettingsPage.h

```diff
@@ -499,6 +499,7 @@ class UserSettingsModel : public QAbstractListModel
         ButtonsInTimeline,
         TypingNotifications,
         ReadReceipts,
+        HiddenTimelineEvents,
```

Comment:
This feels like the wrong section for this. You are configuring a server side setting, but all of the above are client side settings. Maybe it should be its own section?

Reply:
To me this felt like the best place since it has to do with the timeline stuff, but I agree that maybe it's a bit offtopic for that section. I'm not sure where would be a better place to put it though.

Reply:
New section and also put event expiration there?

Reply:
Any suggestions for a section name? Maybe "Global room settings" or something similar?

Reply:
Synchronized timeline settings? Not sure, naming stuff is hard :D

Global message visibility?